recipe/deactivate.sh

```diff
@@ -0,0 +1,4 @@
+if [[ "$__CONDA_OPENSLL_CERT_FILE_SET" == "1" ]]; then
+    unset SSL_CERT_FILE
+    unset
```

Comment:
Should this be `unset __CONDA_OPENSLL_CERT_FILE_SET`?

Reply:
Oups, yes.